sdk/keyvault/azcertificates/utils_test.go

```diff
@@ -70,6 +70,10 @@ func TestMain(m *testing.M) {
 		if err != nil {
 			panic(err)
 		}
+		err = recording.AddHeaderRegexSanitizer("WWW-Authenticate", "https://local", `resource="(.*)"`, &recording.RecordingOptions{GroupForReplace: "1"})
```

Comment:
Nit: the parens shouldn't be necessary since the asterisk only modifies the preceding character i.e., ".".

Reply:
I'm using the parens for grouping so I can replace only the resource URL.